field.go

```diff
@@ -972,14 +988,26 @@ func (f *Field) Value(columnID uint64) (value int64, exists bool, err error) {
 
 // SetValue sets a field value for a column.
 func (f *Field) SetValue(columnID uint64, value int64) (changed bool, err error) {
-	// Fetch bsiGroup and validate value.
+	// Fetch bsiGroup.
 	bsig := f.bsiGroup(f.name)
 	if bsig == nil {
 		return false, ErrBSIGroupNotFound
-	} else if value < bsig.Min {
-		return false, ErrBSIGroupValueTooLow
-	} else if value > bsig.Max {
-		return false, ErrBSIGroupValueTooHigh
+	}
+
+	// Increase max value if the unsigned value is greater.
+	uvalue := uint64(value)
+	if value < 0 {
+		uvalue = uint64(-value)
+	} else if uvalue > uint64(bsig.Max) {
```

Comment:
if `value` is negative, we skip the check here? The comment above seems to indicate this is not correct.

Reply:
yeah, i don't think we want that `else`, in this case?